packages/client/src/components/atoms/CustomerCard/CustomerCard.tsx

```diff
@@ -70,10 +70,15 @@ const CustomerCard: React.FC<CustomerCardProps> = ({
             // if we're rendering category we're applying special formating as
             // translations of multi-word categories are "-" separeted lowercased words
             property === "category"
-              ? capitalizeFirst(t(CategoryLabel[customer.category])).replace(
-                  /-/g,
-                  " "
-                )
+              ? customer.category.map((cat, i) => {
+                  const separator =
+                    i === customer.category.length - 1 ? "" : ", ";
+
+                  return `${capitalizeFirst(t(CategoryLabel[cat])).replace(
+                    /-/g,
+                    " "
```

Comment:
You can omit whitespace and separator by simply adding `categories.map(...).join(", ")` and that handles the first/middle/last element check for you.